classes/solid/telescoping/telescoping.gd

```diff
@@ -56,9 +56,9 @@ func set_disabled(val):
 		safety_net = $SafetyNet
 	safety_net.monitoring = !val
 	
-	var carrier_node = get_node(get_path())
```

Comment:
It was like this for a reason. I don't recall why. Either way, this can be made obsolete by standardizing the way the `disabled` flag works, so that we don't need to do this check.

Reply:
Same method as #291 seems like a good choice. 

If we're talking standardizing, I'd like to discuss making disabling a bit more granular--so the player always knows they're checking "disable collision" or "disable movement" or "disable interaction" rather than wondering, "does disabling a pipe make it non-solid, or just non-enterable?"